apps/snitch_core/lib/core/data/model/stock/stock_movement.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Snitch.Data.Model.StockMovement do
+  @moduledoc """
+  """
+
+  use Snitch.Data.Model
+  alias Snitch.Data.Schema.StockItem, as: StockItemSchema
+  alias Snitch.Data.Schema.StockMovement, as: StockMovementSchema
+
+  @spec create(integer, String.t()) :: {:ok, Ecto.Schema.t()} | {:error, Ecto.Changeset.t()}
+  def create(quantity, stock_item_id) do
```

Comment:
There's no way to set `originator_id`, `originator_type` and `action` field via this method. This module has no docs.

Reply:
Keep it as is, until I start logical parts of it. Since It will be needing originator type, action, originator id logic. Currently implemented basic crud for fields required by spree. They never set these fields when creating. It will be a separate feature, related to stock handling(coordinator, packages etc).